test/stripe.spec.js

```diff
@@ -17,6 +20,35 @@ describe('Stripe Module', function() {
   const cleanup = new testUtils.CleanupUtility();
   this.timeout(20000);
 
+  describe('config object', () => {
+    it('should only accept an object', () => {
+      expect(() => {
+        /* eslint-disable new-cap */
+        Stripe(testUtils.getUserStripeKey(), 'latest');
```

Comment:
If you do that you get the `Do not use 'new' for side effects.eslint(no-new)` error. And if you assign it to a variable you get the `no-unused-variable` error, so either way we need to tell eslint to shut up for a bit 😄

Reply:
heh, sad makes sense.
Might actually be best to disable the `no-new` rule for the whole file, since it's unlikely to be misused and we need the same thing many times in this file